app/_gateway_entities/vault.md

```diff
@@ -112,6 +112,74 @@ features:
     supports_konnect: true
 {% endfeature_table %}
 
+## What can be stored as a secret?
+
+You can store and reference the following fields as secrets in a Vault:
+
+{% feature_table %}
+item_title: Field type
+columns:
+  - title: {{site.konnect_short_name}} Config Store
+    key: konnect
+  - title: Env variable vault
+    key: env
+  - title: HashiCorp Vault
+    key: hcv
+  - title: Google Cloud
+    key: gcp
+  - title: AWS
+    key: aws
+  - title: Azure
+    key: azure
+
+features:
```

Comment:
I don't really understand vaults as well as you do so it might be helpful to ask Lena, but to me it seems like what we are looking for is a table of what **can't** be stored as a secret in the konnect config store, and two bullets that say licenses and the kong.conf values. Then you can do something like: 

* kong.conf values: You can’t reference secrets stored in a [Konnect Config Store](https://deploy-preview-385--kongdeveloper.netlify.app/how-to/configure-the-konnect-config-store/) Vault in kong.conf because Konnect resolves the secret after Kong Gateway connects to the control plane.


Because otherwise it looks like it supports everything but that.

Reply:
Good call out. I think we were also lacking more details about what fields were allowed, so I'll make it a list instead of a feature table and provide a better note about Config Store limitations.

Reply:
List works fine too. Those are all the allowed values 😅 the DP certificates should never have come up - that was an error - but the rest is the accurate list.